_scripts/instruction-widget/install.js

```diff
@@ -131,18 +137,24 @@ module.exports = function(context) {
     context.cbauto = false;
     if (context.webserver == "apache") {
       context.package = "certbot-apache";
+    } else if (context.webserver == "nginx") {
+      context.certonly = true;
```

Comment:
it looks like it's called certbot-nginx on gentoo https://packages.gentoo.org/packages/app-crypt/certbot-nginx